### PR TITLE
Manual syncing from red-hat-storage/ceph-csi:devel into release-4.16

### PR DESCRIPTION
**Pull request by nixpanic**
Pull-Request #270 introduced a conflict that the resync automation job
could not address. This manual merge should make it possible for the
automation to continue again.

Closes: #279 #280

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -166,6 +167,10 @@ cephcsi: check-env
 e2e.test: check-env
 	go test $(GO_TAGS) -mod=vendor -c ./e2e
 
+.PHONY: rbd-group-snapshot
+rbd-group-snapshot:
```
Comment (Madhu-1) [inline Makefile:171]: @nixpanic i hope this is not required

Comment (nixpanic) [reply Makefile:171]: no, that is part of testing I'm doing. But it seems to be included in the devel branch too?

Comment (Madhu-1) [reply Makefile:171]: i think it got in as part of this PR https://github.com/ceph/ceph-csi/pull/4473/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R169 which we should not have done

**Comment by Madhu-1** [comment]
Added label manually as we are still not required to have BZ to sync to release-4.16 branch

**Pull request merged**